Update User Guide For Fall2024 (#336)
project-chip/certification-tool: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

 * limitations under the License.
////

:ubuntu-version: 24.04
:ubuntu-description: Ubuntu Server {ubuntu-version} LTS (64-bit)

> :th-version: Fall 2024

Reviewer: We need to confirm the version string... but I think it should be
```suggestion
:th-version: v2.11-beta2+fall2024
```

| 31          | 10-Jun-2024  | [Apple]Hilton Lima                  | * Updated Test Harness links.
| 32          | 17-Jun-2024  | [Google] Tennessee Carmel-Veilleux  | * Added an example of an external operational dataset.
| 33          | 01-Aug-2024  | [Apple] Carolina Lopes              | * Added information about Certification Mode.
| 34          | 02-Aug-2024  | [Apple] Antonio Melo Jr.            | * Update the Ubuntu mentions to the current supported version.
| 35          | 06-Aug-2024  | [Apple] Antonio Melo Jr.            | * Add the TH release version to the front page of the document. +
                                                                     * Add a warning to TH update when using a different Ubuntu version. +

>                                                                      * Update the OTBR start script parameter. +

Reviewer: ```suggestion
                                                                     * Update the OTBR start script parameter.
```

For hobby developers who want to get acquainted with certification tools/process/TC’s, can spin DUT’s using the example apps provided in the SDK. Refer to the instructions to set up one https://groups.csa-iot.org/wg/members-all/document/folder/3661[here].

> The TH runs on the official *{ubuntu-description}* version. If the TH device happens to be using a different Ubuntu release or other OS, we strongly recommend to fresh install the suggested {ubuntu-description} for reliable results.

Reviewer 2: ```suggestion
The TH runs on the official *{ubuntu-description}* version. If the TH device happens to be using a different Ubuntu release or other OS, we strongly recommend fresh installing version {ubuntu-description} for reliable results.
```
